web/components/AuthenticityMetadata.vue

```diff
@@ -11,21 +11,48 @@
       </template>
 
       <UTable
+        :columns="[
+          { label: '', key: 'label' },
+          { label: '', key: 'value' },
+        ]"
         :rows="[
-          { label: 'Original Recording By', value: recordedByName || 'Unknown' },
-          { label: 'Edited and Published By', value: editedByName || 'Unknown' },
-          { label: 'Transcoded By', value: transcodedByName || 'Unknown' },
-          { label: 'Fingerprint', value: fileCid },
-          { label: 'Registration Time', value: registrationTime },
+          {
+            label: 'Original Recording By',
+            value: recordedByName || 'Unknown',
+            skeletonWidth: 70,
+          },
+          {
+            label: 'Edited and Published By',
+            value: editedByName || 'Unknown',
+            skeletonWidth: 200,
+          },
+          {
+            label: 'Transcoded By',
+            value: transcodedByName || 'Unknown',
+            skeletonWidth: 280,
+          },
+          {
+            label: 'Fingerprint',
+            value: fileCid,
+            skeletonWidth: 400,
+          },
+          {
+            label: 'Registration Time',
+            value: registrationTime,
+            skeletonWidth: 160,
+          },
         ]"
         :ui="{ th: { base: 'hidden' } }"
       >
         <template #label-data="{ row }">
           <span class="font-bold font-mono">{{ row.label }}</span>
         </template>
         <template #value-data="{ row }">
+          <template v-if="isLoadingC2PAManifestStore">
+            <USkeleton class="h-4" :style="`width: ${row.skeletonWidth}px`" />
```

Comment:
Not sure if we want to display loading instead of `Missing C2PA Data` and `Missing numbers data` too

Reply:
Updated
<img width="871" alt="image" src="https://github.com/likecoin/register-c2pa-video/assets/17264716/55152d51-5d22-4f24-8b66-eaa28c4677fe">